src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$cell-size: 100px;
+$cells-per-line: 7;
+$cells-gap: 1px;
+$calendar-padding: 10px;
+$translate-length: 20px;
```

Comment:
Consider using a more descriptive name for the translate length, such as $hovered-cell-translate-length.